charts/kong/README.md

```diff
@@ -253,6 +253,8 @@ section of `values.yaml` file:
 | image.tag                          | Version of the ingress controller                                                     | 0.7.0                                                                        |
 | readinessProbe                     | Kong ingress controllers readiness probe                                              |                                                                              |
 | livenessProbe                      | Kong ingress controllers liveness probe                                               |                                                                              |
+| serviceAccount.create              | Create Service Account for IngresController                                           | true
```

Comment:
"ingress controller" rather than "IngresController" to remain consistent with similar descriptions.